sale_financial_risk_info/models/sale.py

```diff
@@ -9,6 +9,10 @@ class SaleOrder(models.Model):
     _inherit = "sale.order"
 
     risk_info = fields.Html(compute="_compute_risk_info")
+    risk_remaining_value = fields.Monetary(related="partner_id.risk_remaining_value")
+    risk_remaining_percentage = fields.Float(
+        related="partner_id.risk_remaining_percentage"
+    )
```

Comment:
You can use this new fields in _compute_risk_info:
https://github.com/OCA/credit-control/pull/207/files#diff-28133c1098af2cfe22bfacc1d6e5d779a8803bfa2c87d128d54a04f1ff81063dL26

https://github.com/OCA/credit-control/pull/207/files#diff-28133c1098af2cfe22bfacc1d6e5d779a8803bfa2c87d128d54a04f1ff81063dR43-R48

Reply:
risk_available and risk_remaining_percentage have different formulas.

Reply:
Ok, in that case you can include the risk_remaining_percentage parameter in `format info_pattern.format(`

risk_available and risk_remaining_value have the same operation `record.credit_limit - record.risk_total`

This is just a suggestion, I already approved the PR